tests/integration/Db/Dialect/Mysql/ModifyColumnCest.php

```diff
@@ -23,7 +23,7 @@ class ModifyColumnCest
      * Tests Phalcon\Db\Adapter\Pdo\Mysql :: modifyColumn()
      *
      * @author Phalcon Team <team@phalcon.io>
-     * @since  2020-02-27
+     * @since  2020-05-02
```

Comment:
```suggestion
     * @since  2020-02-27
     * @since  2020-05-02 Changed default null and nullable column definition
```

For more see: https://docs.phpdoc.org/latest/references/phpdoc/tags/since.html